integration_tests/src/main/python/benchmark.py

```diff
@@ -124,7 +130,16 @@ def main():
             cmd.append("--summary-file-prefix " + summary_file_prefix)
 
             if args.gc_between_runs is True:
-                cmd.append("--gc-between-runs ")
```

Comment:
There is no need to add a whitespace after each argument since we are building a list which we later turn into a whitespace-separated string